Tests/ML/histopathology/models/test_deepmil.py

```diff
@@ -108,9 +109,12 @@ def test_lightningmodule(
     assert preds.shape[0] == batch_size
 
     for metric_name, metric_object in module.train_metrics.items():
-        if (batch_size > 1) or (not metric_name == "auroc"):
+        if metric_name == MetricsKey.CONF_MATRIX or metric_name == MetricsKey.AUROC:
```

Comment:
the deleted line has a NOT "auroc", were you intentionally trying to change the logic of  the test?

Reply:
Not, that was not the intention. Previously it would run the assert for all metrics apart from AUROC. Now we're getting the same behaviour because the loop continues for all metrics apart from AUROC.